### PR TITLE
Learn more

### LINKED ISSUE #104: Home Page Learn More Button
Follows #105 

**Description**
When user clicks `Learn More` on a recipe card, the user should be redirected to the recipe form page. The recipe form should not have the option to edit because these recipes cards are not coming from the cookbook. The recipe should instead have an option to add it to a cookbook. The recipe form should also display the information from the recipe card. If you feel stuck on the displaying information part, feel free to turn it into another issue or consult with Minh as he constructed most of the API interface.

**Issue Points**: 2

### PR DESCRIPTION
**Pull request by YelllowBelly**
Resolves #104

### CODE REVIEW

File: source/scripts/app.js [hunk 1]
```diff
@@ -439,6 +413,58 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
...
+    author: "author",
+    cuisines: ["cuisine-0", "cuisine-1"],
+    readyInMinutes: 10,
+    image: "/source/images/pasta.jpg",
+    description: "description",
+    ingredients: ["ingredient-1", "ingredient-2"],
+    instructions: ["instruction-1", "instruction-2"],
+  };
+
+  //TODO change dummy to recipes[i] once needed
```
Comment (grantDuntugan) [inline source/scripts/app.js:457]: Is this comment supposed to say "change dummy to recipeCards[i]?

File: source/scripts/app.js [hunk 2]
```diff
@@ -439,6 +413,58 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
...
+  for (let i = 0; i < recipeCards.length; i++) {
+    let button = recipeCards[i].shadowRoot.getElementById("recipe-info-button");
+
+    button.addEventListener("click", populateRecipePage(dummy, true));
+    button.addEventListener("click", () => {
```
Comment (grantDuntugan) [inline source/scripts/app.js:462]:
```suggestion
    button.addEventListener("click", () => {
    populateRecipePage(dummy, true);
```

File: source/scripts/app.js [hunk 3]
```diff
@@ -439,6 +413,58 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
...
+    description: "description",
+    ingredients: ["ingredient-1", "ingredient-2"],
+    instructions: ["instruction-1", "instruction-2"],
+  };
+
+  //TODO change dummy to recipes[i] once needed
+  for (let i = 0; i < recipeCards.length; i++) {
+    let button = recipeCards[i].shadowRoot.getElementById("recipe-info-button");
+
+    button.addEventListener("click", populateRecipePage(dummy, true));
```
Comment (grantDuntugan) [inline source/scripts/app.js:461]:
```suggestion

```

Comment (grantDuntugan) [reply source/scripts/app.js:461]: Does combining the two event listeners work?

Comment (YelllowBelly) [reply source/scripts/app.js:461]: I kept getting a jshint error when I combined both, so I just separated them into two. The functionality is still the same

**Review by grantDuntugan — DISMISSED** [review]

**Review by keli214 — DISMISSED** [review]

**Review by keli214 — APPROVED** [review]

**Review by grantDuntugan — APPROVED** [review]

**Pull request merged**